resources/migrations/0108_auto_20230711_1036.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.11 on 2023-07-11 07:36
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('resources', '0107_auto_20230609_0834'),
```

Comment:
For clarity it would be best to use `--name` argument with `makemigrations` to avoid these kinds of non-descriptive file names ☺️